docs/player_shapeshift_model.md

```diff
@@ -0,0 +1,181 @@
+
+**The \`player_shapeshift_model\` table**
+
+This table holds the information on what values are used for the druid shapeshift models, based on the shapeshift, race, character customisation and their gender.
+
+**Table Structure**
+
+| Field                              | Type    | Attributes | Key | Null | Default | Extra | Comment |
+| ---------------------------------- | ------- | ---------- | --- | ---- | ------- | ----- | ------- |
+| ShapeshiftID (#shapeshiftid)       | TINYINT | UNSIGNED   | PRI | NO   |         |       |         |
+| RaceID (#raceid)                   | TINYINT | UNSIGNED   | PRI | NO   |         |       |         |
+| CustomizationID (#customizationid) | TINYINT | UNSIGNED   | PRI | NO   |         |       |         |
+| GenderID (#genderid)               | TINYINT | UNSIGNED   | PRI | NO   |         |       |         |
+| ModelID (#modelid)                 | INT     | UNSIGNED   |     | NO   |         |       |         |
+
+**Description of the fields**
+
+### ShapeshiftID
+
+| FormID |   Description    |
+| :----: | :--------------: |
+|   1    |     Cat Form     |
+|   5    |    Bear Form     |
+|   8    |  Dire Bear Form  |
+|   27   | Epic Flight Form |
+|   29   |   Flight Form    |
+
+### RaceID
+
+| RaceID | Description |
+| :----: | :---------: |
+|   4    |  Night Elf  |
+|   6    |   Tauren    |
+
+You also may refer to [chrraces](chrraces) "ID" column.
+
+### CustomizationID
+
+If you're a Night Elf it's works by using the [haircolor](characters#haircolor) as the value.
+
+Example below:
+
+| ShapeshiftID | RaceID | CustomizationID [[haircolor](characters#haircolor)] | GenderID | ModelID |                    Comment                    |
+| :----------: | :----: | :-------------------------------------------------: | :------: | :-----: | :-------------------------------------------: |
+|      5       |   4    |                          6                          |    2     |  29414  | Night Elf, any gender, bear colour: Dark Blue |
+
+If you're a Tauren it's works by using the [skin](characters#skin) as the value.
+
+| ShapeshiftID | RaceID | CustomizationID [[skin](characters#skin)] | GenderID | ModelID |                 Comment                 |
+| :----------: | :----: | :---------------------------------------: | :------: | :-----: | :-------------------------------------: |
+|      5       |   6    |                     0                     |    0     |  29418  | Tauren, male, bear colour: Dark (Black) |
+|      5       |   6    |                     2                     |    1     |  29419  |   Tauren, female, bear colour: White    |
+
+### GenderID
+
+| [GenderID](characters#gender) | Description |
+| :---------------------------: | :---------: |
+|               0               |    Male     |
+|               1               |   Female    |
+|               2               |   Unknown   |
+
+### ModelID
+
+Refer to [creature_model_info](#creature_model_info#displayid)
+
+### Default Values
```

Comment:
As mentioned in Discord, I don't think this table really warrants having its values emplaced in the wiki page.